content/doc/book/system-administration/systemd-services.adoc

```diff
@@ -83,13 +91,31 @@ Environment="JENKINS_OPTS=--prefix=/jenkins --javaHome=/opt/jdk-11"
 Environment="CASC_JENKINS_CONFIG=/var/lib/jenkins/configuration-as-code/"
 ----
 
+WARNING: `systemctl edit jenkins` creates the drop-in unit as `root` with 0644 (`-rw-r--r--`) permissions.
+The migration logic, on the other hand, creates the drop-in unit as `root` with 0600 (`-rw-------`) permissions.
+This might be of consequence if you store an HTTPS keystore location and/or password in the drop-in unit
+and also run jobs directly on the controller,
+a practice which the Jenkins project link:https://www.jenkins.io/doc/book/security/controller-isolation/[explicitly discourages].
+When in doubt, secure the drop-in unit by settings its permissions to 0600 with `chmod(1)`.
+
+The drop-in unit unifies configuration across all three distributions: Debian, Red Hat, and openSUSE.
+Also note that the drop-in unit is not overwritten on upgrades.
+
+NOTE: Unlike the System V `init(8)` configuration, the `override.conf` file only contains customizations, not the original defaults.
+Users who are accustomed to editing an existing set of defaults must refer to the (read-only) service unit side-by-side when editing the drop-in unit
+or use a command like `systemctl edit jenkins --full`, which copies the original service unit instead of creating a drop-in unit.
+
+A final point to mention about the service unit is its use of specifiers,
+which may be unfamiliar to some users.
+The `systemd(1)` documentation contains link:https://www.freedesktop.org/software/systemd/man/systemd.unit.html#id-1.13.3[a table of specifiers available in unit files].
```

Comment:
A proposal for a slightly broader description of a specifier and why it might be useful or interesting to an administrator.
```suggestion
A final point to mention about the service unit is its use of specifiers,
which may be unfamiliar to some users.
The drop-in unit does not perform shell expansion.
Specifiers can insert contextual information (like system hostname, unit name, and operating system kernel release) into the drop-in unit.
The `systemd(1)` documentation contains link:https://www.freedesktop.org/software/systemd/man/systemd.unit.html#id-1.13.3[a table of specifiers available in unit files].
```